js/widgets/popup.arrow.js

```diff
@@ -29,11 +29,16 @@
 	}
 } )( function( $ ) {
 
-var ieHack = ( $.mobile.browser.oldIE && $.mobile.browser.oldIE <= 8 ),
-	uiTemplate = $(
-		"<div class='ui-popup-arrow-guide'></div>" +
-		"<div class='ui-popup-arrow-container" + ( ieHack ? " ie" : "" ) + "'>" +
-		"<div class='ui-popup-arrow'></div>" +
+var uiTemplate = $(
```

Comment:
Constructing the container this way no longer seems necessary or like the best choice it actually makes it less obvious what the intended structure is now. If you construct it detached and using append you can save references and add classes as you build instead of doing selection.